Kickstarter-iOS/ViewModels/AppDelegateViewModel.swift

```diff
@@ -401,6 +404,20 @@ AppDelegateViewModelOutputs {
       .filter { $0 == .tab(.login) }
       .ignoreValues()
 
+    self.goToCreatorMessageThread = deepLink
+      .map { navigation -> (Param, Int)? in
+        guard case let .creatorMessages(projectId, messageThreadId) = navigation else { return nil }
+        return .some((projectId, messageThreadId: messageThreadId))
+      }
+      .skipNil()
+      .switchMap { projectId, messageThreadId in
+        AppEnvironment.current.apiService.fetchMessageThread(messageThreadId: messageThreadId)
+          .demoteErrors()
+          .map { (env: MessageThreadEnvelope) in
```

Comment:
Was this causing ambiguity as just as ` .map { env in ... }`? Just curious why the type is specified.